packages/reports/addon/components/common-actions/schedule.ts

```diff
@@ -290,7 +290,9 @@ export default class ScheduleActionComponent extends Component<Args> {
     assert('The localDeliveryRule is defined', this.localDeliveryRule);
     assert('The format is defined', this.localDeliveryRule.format);
     this.localDeliveryRule.format.type = type;
-    if (this.localDeliveryRule.name.toString().match(/\w+ \bdelivered\b +\w+ \bevery\b +\w/)) {
+    if (
+      this.localDeliveryRule.name.toString().match(/\w+ \bdelivered\b +\w+ \bevery\b (day|week|month|quarter|year)$/)
```

Comment:
This is still acting funky - for example, if you add a number to the end of the default name, for example, the next time you open the modal it only have the default name and not the number the user added

Reply:
That's because I had something in onOpen that was replacing it, its working fine now